app/src/pages/forgotPasswordPage/forgotPasswordPage.tsx

```diff
@@ -74,19 +77,24 @@ export const ForgotPasswordPage = (props: IForgotPasswordPageProps): React.React
   const renderEmailSentForm = (): React.ReactElement => {
     return (
       <Box variant='card' isFullWidth={false}>
-        <Text variant='header3'>Recover Password</Text>
-        <Spacing variant={PaddingSize.Wide} />
-        <Text>An email has been sent. Please click the link when you get it.</Text>
+        <Stack shouldAddGutters={true} direction={Direction.Vertical} childAlignment={Alignment.Fill}>
+          <Text variant='header2' alignment={TextAlignment.Center}>Recover Password</Text>
+          <Spacing variant={PaddingSize.Wide} />
+          <Text>An email has been sent. Please click the link when you get it.</Text>
+        </Stack>
       </Box>
     );
   };
 
   return (
     <ContainingView>
-      <Spacing variant={PaddingSize.Wide} />
-      <Stack isFullHeight={true} isFullWidth={true} shouldAddGutters={true} direction={Direction.Vertical} childAlignment={Alignment.Center}>
-        {hasRecoverPasswordFormBeenSubmitted ? renderEmailSentForm() : renderRecoverPasswordForm()}
-      </Stack>
+      <ResponsiveContainingView sizeResponsive={{ base: 12, small: 8, medium: 6, large: 5 }}>
```

Comment:
what does base mean, what do these numbers mean?

Reply:
its for responsiveness. base is i think <560px (or something close) small is 560-760 (or something close) and so on. it just makes sure the box stays looking nice in multiple sizes (copied from everypage console)